docs/content/developer/network-overview.mdx

```diff
@@ -18,6 +18,9 @@ In general, all endpoints are load-balanced but also subject to rate limiting. I
 ## IOTA Mainnet
 
 <NetworkInfo.L1 {...Networks['iota']}/>
+:::info
+[Nirvana Labs](https://nirvanalabs.io/) provides hosted RPC nodes, archival services, and indexers to support developers and applications building on IOTA Rebased.
```

Comment:
`IOTA Rebased` is over. It is now the `IOTA Mainnet`

Reply:
Good point. We should check this all over the docs

Reply:
https://github.com/iotaledger/iota/issues/6911